rad/src/error.rs

```diff
@@ -253,11 +277,251 @@ pub enum RadError {
         achieved, required
     )]
     InsufficientConsensus { achieved: f64, required: f64 },
+    /// The request contains too many sources.
+    #[fail(display = "The request contains too many sources")]
+    RequestTooManySources,
```

Comment:
Do we want to add arguments to this errors? For example `RequestTooManySources { num_sources: u8 }`.

Reply:
Should be great, like num_sources and limit_sources

Reply:
Oh nevermind, they are not implemented yet. The arguments will be added when they are implemented